sdk/cosmosdb/cosmos/src/client/ChangeFeed/ChangeFeedRetentionTimeSpan.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/*
+ * Represents the change feed policy configuration for a container in the Azure Cosmos DB service.
+ */
+export class ChangeFeedRetentionTimeSpan {
+  private retentionInMinutes: number;
+
+  constructor(minutes: number) {
+    if (typeof minutes !== "number") {
```

Comment:
What are the chances that this will not be a "number" if it is defined as "number"?

Reply:
Defining it as `number` will only work for TS. This extra check is for devs using JS.